docs/tutorials/index.rst

```diff
@@ -0,0 +1,49 @@
+GeoMesa |release| Tutorials
+===========================
+
```

Comment:
The tutorials are already available: https://github.com/geomesa/geomesa-tutorials
But I think we need to review them to make sure they still work, don't require e.g. geomesa-ext (or have instructions for that), etc.